### DISCUSSION TITLE
Top-level vs. sub-component licenses

### DISCUSSION THREAD

**Discussion by aboncok** [qos-ch/slf4j#315] [Q&A] [answered]
Hi,
Most sub-components have their own separate License.txt, which is in most cases 13 years old. In fact, those are Apache licenses, while the top-level is MIT.
So, which is the active one?

**Comment by ceki** [comment]
Hi @aboncok, 

All modules are MIT licensed except for jcl-over-slf4j module which is Apache Licensed. Do you observe otherwise?

**Reply by aboncok** [reply]
My bad and you are right.
Is that intentional or it just stuck with it?

Also, a few components' copyright text is not updated. Not a huge deal probably, but the year still shows 2004-2007.

**Comment by ceki** [comment, marked as answer]
The jcl-over-slf4j is derived from preexisting Apache Licensed code, so we respect the existing license.

Thank you for the heads up regarding the license year. I'll fix it.

**Reply by aboncok** [reply]
Thanks!
